README.md

```diff
@@ -3,8 +3,28 @@
 Welcome! Are you interested in contributing to one of CNCF hosted projects? This repository 
 should help you contains information and guidelines about contributions to CNCF projects.
 
+CNCF offers multiple ways to start contributing to the CNCF ecosystem, including either foundation-wide and project-wide opportunities.
+
+# TOC
+
+The CNCF TOC is the technical governing body of the CNCF Foundation. The recommended way to start contributing to CNCF TOC - acting as a TOC Contributor. Details - [here].(https://github.com/cncf/toc/blob/master/CONTRIBUTORS.md)
+
+## Working Groups
+
+The TOC has created the working groups (WG) to investigate and discuss the following topics:
+
+| Working Group | Chair            | Meeting Time                          | Minutes/Recordings |
+|---------------|------------------|---------------------------------------|--------------------|
+| [CI](https://github.com/cncf/wg-ci)         | Camille Fournier | [4th Tue of every month at 8AM PT](https://zoom.us/my/cncfciwg) | [Youtube](https://www.youtube.com/playlist?list=PLj6h78yzYM2P3_A3ujWHSxOu1IO_bd7Zi)
+| [Networking](https://github.com/cncf/wg-networking) | Ken Owens        |  [1st and 3rd Tue every month at 9AM PT](https://zoom.us/my/cncfnetworkingwg)                                      | [Youtube](https://www.youtube.com/playlist?list=PLj6h78yzYM2M_-K5n67_zTdrPh_PtTKFC)
+| [Serverless](https://github.com/cncf/wg-serverless) | Ken Owens        |  [Thu of every week at 9AM PT](https://zoom.us/my/cncfserverlesswg)                                     | [Youtube](https://www.youtube.com/playlist?list=PLj6h78yzYM2Ph7YoBIgsZNW_RGJvNlFOt)
+| [Storage](https://github.com/cncf/wg-storage)    | Ben Hindman      |  [2nd and 4th Wed every month at 8AM PT](https://zoom.us/my/cncfstoragewg)                                     | [Youtube](https://www.youtube.com/playlist?list=PLj6h78yzYM2NoiNaLVZxr-ERc1ifKP7n6)
+
+
 # Projects
 
+The Projects of the Cloud Native Computing Foundation are listed below, together with the brief information on engaging with them:
```

Comment:
describe the different level of projects, how to bring a project to CNCF etc, no need to list every single one, you can link to the CNCF TOC repo for projects or the CNCF landscape (which needs its own section)

Reply:
> no need to list every single one

I don't agree. Here we have a contribution-specific information, while CNCF TOC and CNCF Landscape provide the project overviews.

If README.md here looks too overloaded because of the project descriptions, I'd propose moving this data to a separate file, but not destroying it entirely.

cc @mrbobbytables